README.rst

```diff
@@ -233,6 +243,10 @@ Usage
        -j, --json            Display dependency tree as json. This will yield "raw"
                              output that may be used by external tools. This option
                              overrides all other options.
+       --output OUTPUT_FORMAT
```

Comment:
I think `--graph-output` would be a better name making it explicit that it will result in a graph related output